wgsl/index.bs

```diff
@@ -22,9 +22,11 @@ Ignored Vars: i, c0, e, e1, e2, e3, edge, eN, p, s1, s2, sn, AS, AM, N, newbits,
 !Participate: <a href="https://github.com/gpuweb/gpuweb/issues/new?labels=wgsl">File an issue</a> (<a href="https://github.com/gpuweb/gpuweb/issues?q=is%3Aissue+is%3Aopen+label%3Awgsl">open issues</a>)
 !Tests: <a href=https://github.com/gpuweb/cts/tree/main/src/webgpu/shader/>WebGPU CTS shader/</a>
 
+Editor: Alan Baker, Google https://www.google.com, alanbaker@google.com
```

Comment:
@alan-baker do you have a W3C ID?

( The only way know how to find it is from the URL of my w3C member page.)

Reply:
Ah.  I can get it from the Google participants page. https://www.w3.org/organizations/35662/participants (thanks @dj2)